crates/services/p2p/src/service.rs

```diff
@@ -568,6 +613,13 @@ pub mod tests {
             }))
         }
 
+        fn get_sealed_headers_range(
+            &self,
+            _block_height_range: Range<u32>,
+        ) -> StorageResult<Vec<SealedBlockHeader>> {
+            todo!()
```

Comment:
Do we want to fix that during this PR?=)

Reply:
This is parity with what we had before. I'd be happy to do the work to get tests working for all the requests but should probably be done in a separate Issue if we want to do it correctly.

I actually wrote down some notes about this. I think this is the correct interface to test, instead of those other  tests where we manually assemble a response. I call this out at the beginning of this PR ^^^.. I'm pretty sure it would take some modifications to `Task` to make the `p2p_service` mockable (at least that's what I have in my notes, we'd have to double-check).

Reply:
Could you use `unreachable` instead, please?

Hmm, it seems we don't test any logic except start of the service. Could you update all methods in the implementation of the `P2pDb` trait to do `unreachable!()`?

Reply:
`unreachable!()` wouldn't be correct. I can just call the methods directly. How about `unimplemented!()`?